bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -3,6 +3,7 @@
 load '../helpers/load'
 
 local_setup() {
+    RD_RAMDISK_SIZE=$((15 * 1024)) # We need more disk to run the Rancher image.
```

Comment:
Why do we need 15GB here, but only 12GB in `bats/tests/containers/run-rancher.bats`?

Also

```suggestion
    needs_ramdisk_size 15
```

Reply:
Typo; made both do 12.